src/Controller/ComptableController.php

```diff
@@ -33,6 +34,30 @@ public function index(InvoiceRepository $invoiceRepository, Request $request, Cl
 
             $invoices = $invoiceRepository->findInvoicesCreatedAfterDate($date->date);
             $invoicesPrices = $invoiceRepository->findTotalPriceOfInvoicesCreatedAfterDate($date->date);
+            $productDetails = [];
+
+            foreach ($invoices as $invoice) {
+                $productName = $invoice->getName();
+                if (!isset($productDetails[$productName])) {
+                    // Récupère le produit par son nom
+                    $product = $productRepository->findByProductName($productName);
+                    if ($product) {
+                        // Initialise ou met à jour le compteur pour ce produit
+                        $productDetails[$productName] = [
+                            'product' => $product,
+                            'count' => 1 // Initialisation du compteur pour ce produit
+                        ];
+                    }
+                } else {
+                    // Incrémente le compteur si le produit a déjà été ajouté
+                    $productDetails[$productName]['count']++;
+                }
+            }
+
+            // Trier par nombre de ventes, si nécessaire
+            uasort($productDetails, function ($a, $b) {
+                return $b['count'] <=> $a['count'];
+            });
```

Comment:
Is it not possible to make all this logic with sql directly ? we could use count to count the number of row and order by for sort